testcontainers/src/core/container.rs

```diff
@@ -182,18 +189,23 @@ where
         .unwrap_or_else(|_| panic!("container {} has missing or invalid bridge IP", self.id))
     }
 
-    pub fn exec(&self, cmd: ExecCommand) {
+    pub fn exec(&self, cmd: ExecCommand) -> ExecOutput {
```

Comment:
It is unfortunately still a breaking change but that is okay. Can you add a changelog entry please?

Reply:
Yeah. you're right. Sure